src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -488,22 +495,241 @@ private void testFileWithoutInfoLineInHeader(final String name, final Class<? ex
         spec.executeTest(name, this);
     }
 
-    @Test
-    public void testInvalidJexl() throws IOException {
-        final String testFile = getToolTestDataDir() + "ac0.vcf";
-
+    private void beforeJexlTest(){
         // NOTE: JexlEngine singleton construction in VariantContextUtils sets silent to false.
         // However VariantFiltration.initialize() sets setSilent(true) on the shared instance.
         // Just in case this test runs after a VariantFiltration in the same VM, always set silent back to false.
         htsjdk.variant.variantcontext.VariantContextUtils.engine.get().setSilent(false);
+    }
+
+    @Test // tsato: this test fails for the wrong reason; ac0.vcf does not exist
+    public void testInvalidJexl() throws IOException {
+        final GATKPath testFile = new GATKPath(getToolTestDataDir() + "ac0.vcf");
+        beforeJexlTest();
 
         final IntegrationTestSpec spec = new IntegrationTestSpec(
-                baseTestString(" -select 'vc.getGenotype(\"FAKE_SAMPLE\").isHomRef()' ", testFile),
+                baseTestString(" -select 'vc.getGenotype(\"FAKE_SAMPLE\").isHomRef()' ", testFile.toString()),
                 1,
                 UserException.class);
         spec.executeTest("InvalidJexl", this);
     }
 
+    // If two separate --select command lines are used e.g. --select "'AF > 0.05'" --select "'MQRankSum > 3'"
+    // then the booleans should be combined with the logical-or
+    final GATKPath svHGDBMultiSampleVcf = new GATKPath(getToolTestDataDir() + "hgdb_sv_multi_sample_mini.vcf");
+    @Test
+    public void testMultipleINFOJexls() {
+        beforeJexlTest();
+        // Is File acceptable here? Or use Path? GATKPath?
+        final File testOutput = createTempFile("multiple_jexl_test", "vcf");
+        final Predicate<VariantContext> filter = vc -> vc.getAttributeAsInt("AC", -1) > 0 ||
+                vc.getAttributeAsString("SVTYPE", "").equals("DEL");
+
+        final Pair<VCFHeader, List<VariantContext>> preFilter = VariantContextTestUtils.readEntireVCFIntoMemory(svHGDBMultiSampleVcf.toString());
+        final int expectedCount = (int) preFilter.getRight().stream().filter(filter).count();
+
+        runCommandLine(Arrays.asList("-V", svHGDBMultiSampleVcf.toString(),
+                        "-O", testOutput.getAbsolutePath(),
+                        "--select", "SVTYPE == 'DEL'",
+                        "--select", "AC > 0"),
```

Comment:
Hmm, I am puzzled that this passes, since you changed the long name of the `-select` argument to `--select-expressions` above. Same comment applies to the tests below. Could you consult with @cmnbroad to get his opinion on why `--select` is still being accepted as a long name for the argument?

Reply:
@droazen @takutosato IIRC, the underlying parser (jopt) that we use has the surprising policy that it accepts long and short names interchangeably (it will accept either name for both '-' and '--').

Reply:
Thanks @cmnbroad, good to know